src/schemas/json/workflows.json

```
@@ -259,6 +259,48 @@
         },
         "for": {
           "$ref": "#/definitions/for"
+        },
+        "parallel": {
+          "type": "object",
+          "description": "Run branches or iterations in parallel",
+          "additionalProperties": false,
+          "properties": {
+            "exception_policy": {
+              "description": "determines action of other branches when an exception occurs",
+              "type": "string",
+              "enum": ["continueAll"]
+            },
+            "shared": {
+              "description": "list of shared variables",
+              "type": "array",
```

```suggestion
              "type": "array",
              "uniqueItems": true,
```

Optional fix: missing array constraints: `uniqueItems` can be added to avoid duplicates inside array.